scripts/cc-cli/src/commands/send.ts

```diff
@@ -29,8 +32,27 @@ async function sendAction(options: OptionValues) {
 
   // Check options
   checkAmount(options);
-  checkAddress(options.to, api);
+  checkAddress(options.to, api, "send funds to");
 
+  if (options.scheme === "ecdsa") {
```

Comment:
Maybe use `.toLowerCase()` to be more flexible as to what the user will type. WDYT ?